src/view/utils/index.js

```diff
@@ -172,8 +172,11 @@ export const waitUntil = ( condition, callback, maxTries = 10 ) => {
  * A common scenario is amp-analytics being blocked by ad blockers or privacy browser extensions like uBlock or Ghostery.
  * In this case the custom element will exist but the service's main class will not exist on the global __AMP_SERVICES object.
  * Extensions in this object may or may not be prefixed with `amp-`: e.g. `__AMP_SERVICES.access` vs. `__AMP_SERVICES.amp-analytics`.
+ *
+ * AMP Analytics should always be polyfilled, as non-AMP JS running in AMP Plus can break AMP Analytics features.
```

Comment:
Polyfilling AMP Analytics while some other plugin enqueues AMP Analytics would result in doubled pageviews & other events. And I [can't reproduce the issue](https://github.com/Automattic/newspack-popups/issues/981#issuecomment-1265372140) it's supposed to fix.

Reply:
Ah, that's a good point. We'll have to do some work to ensure that when AMP Plus is on, the Campaigns AMP Analytics JSON is not picked up by the real AMP Analytics if it also happens to be active.

Reply:
5c3b80b reverts the polyfill—we can tackle that in a separate PR.